accounts/pkg/storage/disk.go

```diff
@@ -0,0 +1,170 @@
+package storage
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"github.com/owncloud/ocis/accounts/pkg/config"
+	"github.com/owncloud/ocis/accounts/pkg/proto/v0"
+	olog "github.com/owncloud/ocis/ocis-pkg/log"
+)
+
+var groupLock sync.Mutex
+
+// DiskRepo provides a local filesystem implementation of the Repo interface
+type DiskRepo struct {
+	serviceID string
+	cfg       *config.Config
+	log       olog.Logger
+}
+
+// NewDiskRepo creates a new disk repo
+func NewDiskRepo(serviceID string, cfg *config.Config, log olog.Logger) DiskRepo {
+	paths := []string{
+		filepath.Join(cfg.Repo.Disk.Path, accountsFolder),
+		filepath.Join(cfg.Repo.Disk.Path, groupsFolder),
+	}
+	for i := range paths {
+		if _, err := os.Stat(paths[i]); err != nil {
+			if os.IsNotExist(err) {
+				if err = os.MkdirAll(paths[i], 0700); err != nil {
+					log.Fatal().Err(err).Msgf("could not create data folder %v", paths[i])
+				}
+			}
+		}
+	}
+	return DiskRepo{
+		serviceID: serviceID,
```

Comment:
same as earlier, `serviceID` is not being used.

Reply:
👍